src/components/flowchart/flowchart.js

```diff
@@ -446,11 +446,19 @@ export class FlowChart extends Component {
    * @param {Object} node Datum for a single node
    */
   handleNodeClick = (event, node) => {
-    if (node.type === 'modularPipeline') {
-      this.props.onClickToExpandModularPipeline(node.id);
-    } else {
-      this.props.onLoadNodeData(node.id);
-      this.props.toSelectedNode(node);
+    const { type, id } = node;
+    const {
+      onClickToExpandModularPipeline,
+      displayMetadataPanel,
+      onLoadNodeData,
+      toSelectedNode,
+    } = this.props;
+
+    if (type === 'modularPipeline') {
+      onClickToExpandModularPipeline(id);
+    } else if (displayMetadataPanel) {
+      onLoadNodeData(id);
```

Comment:
on onLoadNodeData calls another action called toggleNodeClicked so we need to add that here because right now when you click the node, the behavior is the same as hover 

```python 
f (type === 'modularPipeline') {
      onClickToExpandModularPipeline(id);
    } else 
      { if (displayMetadataPanel) {
          onLoadNodeData(id);
          toSelectedNode(node);
        }
        else {
          onToggleNodeClicked(id);
        }
      }
    
    event.stopPropagation();
  };
```

You will also need to add that action to mapDispatchtoProps

Reply:
Updated. `onToggleNodeClicked ` update `state.node.click` state so made some changes to `metadata.js` to prevent change in chart size update.